src/CoreShop/Bundle/CustomerBundle/Form/Type/CustomerType.php

```diff
@@ -66,7 +69,14 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                 'invalid_message' => 'coreshop.form.customer.email.must_match',
                 'first_options' => ['label' => 'coreshop.form.customer.email'],
                 'second_options' => ['label' => 'coreshop.form.customer.email_repeat'],
+            ])
+        ;
+
+        if ($options['allow_username']) {
+            $builder->add('username', TextType::class, [
+                'label' => 'coreshop.form.customer.username',
```

Comment:
Thee username field gets appended, so the username input field will show up after the submit button.  Could we change the priority?

Reply:
na - my fault. Everything is fine here. I had my own form layout in twig.